test/e2e-fsgroup/README.md

```diff
@@ -0,0 +1,78 @@
+# FSGroup e2e Tests
+
+## Installing Ginkgo v2
+
+To install Ginkgo, use the following command:
+
+```bash
+go install github.com/onsi/ginkgo/v2/ginkgo
+```
+
+### Verify Installation
+
+After installation, check the Ginkgo version with:
+
+```bash
+/root/go/bin/ginkgo version
+```
+
+You should see output similar to:
+
+```
+Ginkgo Version 2.x.x
+```
+
+Move the Ginkgo binary to a more accessible location:
+
+```bash
+mv /root/go/bin/ginkgo /usr/bin/ginkgo
```

Comment:
Would have been nice if these steps could be part of a Makefile target that checks for ginkgo and downloads and installs the appropriate version.

Reply:
Sure! @donatwork, added Makefile target for checking, downloading ginkgo and running the tests